editor/plugins/editor_preview_plugins.cpp

```diff
@@ -470,10 +470,10 @@ Ref<Texture> EditorScriptPreviewPlugin::generate(const RES &p_from) {
 	if (EditorSettings::get_singleton()->get("text_editor/theme/color_theme") == "Adaptive") {
 		Ref<Theme> tm = EditorNode::get_singleton()->get_theme_base()->get_theme();
```

Comment:
You could remove this line too if the proposed fix is correct.

Reply:
Done !